src/app.py

```diff
@@ -78,13 +88,13 @@ def gather_magazines():
         len_data = len(data)
         parse_counter = 1
         for i in range(1, len_data):
-            if data[i][5] != "1" and data[i][0] != "":
```

Comment:
some more comments here would make this easier for someone to edit this in the future if we want to improve on our system